.gitignore

```diff
@@ -17,3 +17,6 @@
 
 # Ignore uploaded files
 /public/uploads
+
+# Ignote simplecov coverage
+/coverage
```

Comment:
Yes I use Sublime, and already have that extension installed. I assure you I didn't mean to upset any one if any thing it's the other way around :)